src/apps/adop_train.cpp

```diff
@@ -307,6 +307,7 @@ class NeuralTrainer
         torch::set_num_threads(1);
 
         std::string experiment_name = Saiga::CurrentTimeString("%F_%T") + "_" + params->train_params.name;
```

Comment:
Can you please check if changing `"%F_%T"` to `"%F_%H-%M-%S"` also fixes your problem? I think this would be the cleaner solution.

Reply:
Finally got to recompile and test it. Yes, it works fine. Thanks for help!